libbeat/filter/condition.go

```diff
@@ -0,0 +1,330 @@
+package filter
+
+import (
+	"fmt"
+	"reflect"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+)
+
+type EqualsValue struct {
+	Int int
+	Str string
+}
```

Comment:
It's kind of confusing that this one is defined here, but the `RangeValue` is in config.go.  I'd say move one of them near the other.